platform_versioned_docs/version-23.2.0/enterprise/configuration/overview.mdx

```diff
@@ -44,7 +42,9 @@ Specify general Tower configuration values in your environment variables. The bo
 
 `TOWER_CRON_SERVER_PORT`: Define the HTTP port usd by the Tower cron service (default: `8080`, requires Tower 21.06.1 or later).
 
-`TOWER_USER_WORKSPACE_ENABLED` : Enable or disable the showing of the user private workspace context. (default: `true`, requires Tower 22.1.0 or later).
+`TOWER_USER_WORKSPACE_ENABLED`: Enable or disable the showing of the user private workspace context. (default: `true`, requires Tower 22.1.0 or later).
+
+`TOWER_ENABLE_ARM64`: Enable support for ARM64 CPU architectures. Required to enable Graviton in AWS Batch compute environments. 
```

Comment:
I would qualify this with "Currently only available for ..." (probably exclusive to AWS Batch right now).